Java/me/dilley/MineStat.java

```diff
@@ -125,7 +125,7 @@ public boolean refresh()
     else
     {
       serverData = rawServerData.split("\u0000\u0000\u0000");
-      if(serverData != null && serverData.length >= NUM_FIELDS)
+      if(serverData.length >= NUM_FIELDS)
```

Comment:
The null check was here as a "better safe than sorry" way of preventing a `NullPointerException` in the event that the `serverData` object could be `null`. If `serverData` is `null`, accessing the `length` member will throw the exception. However, since `String.split()` does not return `null` per the Javadocs, we should be fine with this change.